rllib/agents/mbmpo/tests/test_mbmpo.py

```diff
@@ -27,7 +27,7 @@ def test_mbmpo_compilation(self):
         for _ in framework_iterator(config, frameworks="torch"):
             trainer = mbmpo.MBMPOTrainer(
                 config=config,
-                env="ray.rllib.examples.env.mbmpo_env.PendulumWrapper")
+                env="ray.rllib.examples.env.mbmpo_env.CartPoleWrapper")
```

Comment:
We should still test both right?

Reply:
Turns out MBMPO is not tested anymore (in MAML higher PR) b/c torch MAML uses higher.